docs/tutorials/writing_a_series.rst

```diff
@@ -0,0 +1,112 @@
+Writing a Series File
+=====================
+
+This is a step-by-step tutorial on how to write a series file in Pavilion.
+
+A Test Series is a group of tests that is created whenever the user invokes
+``pav run <test names>`` or ``pav run -f <file of test names>``. A series file
+allows the user to specify test hierarchies, assign modes to specific tests,
+and dictate other settings for tests which are grouped together in the same
+series. Creating Test Series in this way also gives the user the option of
+running tests in a continuous manner.
+
+.. contents:: Table of Contents
+
+Where to Write Series Files
+---------------------------
+
+Series files are yaml files and are placed in the <pav config dir>/series
+directory.
+
+Series File Elements
```

Comment:
Series Configuration

'Elements' is internal, 'yaml_config' terminology that shouldn't be user facing.